src/containers/my-resources/lesson-container/LessonsContainer.jsx

```diff
@@ -0,0 +1,20 @@
+import AddResourceWithInput from '~/containers/my-resources/add-resource-with-input/AddResourceWithInput'
+import { useRef } from 'react'
+import { useTranslation } from 'react-i18next'
+import { authRoutes } from '~/router/constants/authRoutes'
+
+const LessonContainer = () => {
+  const { t } = useTranslation()
+  const searchTitle = useRef
```

Comment:
wrong usage of useRef hook
Look usage example [here](https://react.dev/reference/react/useRef#usage)